src/index.js

```diff
@@ -0,0 +1 @@
+import './styles.css';
```

Comment:
- [ ] Please check that you use some modules for import and export as per the activity requirement:

![image](https://user-images.githubusercontent.com/99830261/202233475-48527a69-ebd3-4419-be28-4e0fee379b73.png)
I recommend that you make a fixed score list module and a module with a function to display said list, that way you can practice importing and exporting between modules.